src/Controller/HomeController.php

```diff
@@ -13,6 +13,6 @@ class HomeController extends AbstractController
      */
     public function index(): Response
     {
-        return $this->render('home/index.html.twig');
+        return $this->render('home/index.html.twig', ['website' => 'AMON']);
```

Comment:
If it's a constant value, there is no need for a twig variable, just write it in the template

Reply:
Ok done